procmonAnalyze.js

```diff
@@ -15,16 +15,34 @@ const HOVERED_ENTRY_FILL = 0.9;
 const FILTERED_OUT_ENTRY_FILL = 0.7;
 const MAX_DETAIL_LINES = 24;
 const ASSUMED_CLUSTER_SIZE = 4096;
+const TIMELINE_SELECTION_OVERLAY_DEPTH = 0.8;
+const TIMELINE_UNSELECTED_GREY_TOP = `#d6d5d2`;
+const TIMELINE_UNSELECTED_GREY_BOTTOM = `#d6d5d1`;
 
 const csvInput = document.getElementById("csvfile");
 const diskifyInput = document.getElementById("diskifyfile");
 const profilerInput = document.getElementById("profilerfile");
 const tooltip = document.getElementById("tooltip");
 const searchbar = document.getElementById("searchbar-input");
 const colorBySelect = document.getElementById("color-by-select");
+const refocus = document.getElementById("redraw")
+const reset = document.getElementById("reset");
 const timeline = document.getElementById("timeline");
 const canvas = document.getElementById("canvas");
 const fsmapCanvas = document.getElementById("fsmap-canvas");
+const markersCanvas = document.getElementById("canvas-markers");
+var data, diskifyData; // Contains the entire array of data from input file
+
+var offsetX, offsetY;
+reOffset();
+window.onscroll=function(e){ reOffset(); }
```

Comment:
Hmm. I would rather we make sure that the window doesn't scroll, rather than add a scroll handler. Thoughts?